README.md

```diff
@@ -122,6 +122,17 @@ python_version = 3.12
 python_full_version = 3.12.7
 implementation_name = cpython
 
+[env.windows]
+os_name = nt
+platform_tag = win_amd64
```

Comment:
I got this value from trial and error, would you agree that this is the correct one?

Reply:
platform_tag is optional, actually, it may even be better to drop it in this case. I'm trying to look in the "packaging" documentation if they're converting dashes to underscores but I'm not seeing anything right now.

Reply:
After some testing I have determined that not specifying a platform tag or specifying `win-amd64` means all prebuilt Windows packages are skipped.

Specifying `win_amd64` or `(win_amd64|win-amd64)` means the prebuilt ones are included.

Reply:
Great, so this is all good? I can merge?

Reply:
I'll make another commit soon, just testing some last things.

Reply:
It's ready to merge now.